framework/src/org/checkerframework/common/basetype/messages.properties

```diff
@@ -64,6 +64,8 @@ purity.more.deterministic=the method %s could be declared as @Deterministic
 
 flowexpr.parse.index.too.big=the method does not have a parameter %s
 flowexpr.parse.error=cannot parse the expression '%s'
+# Same as above, but only issued at call sites for a method with a postcondition.
+flowexpr.parse.postcondition.error=cannot parse the expression '%s'
```

Comment:
If we were to use `flowexpr.parse.error.postcondition` or `postcondition.flowexpr.parse.error` we would automatically suppress this new warning when suppressing `flowexpr.parse.error`, which I think would be nice.